### PR TITLE
Iht 5139 IHT - Flat content iteration Deadlines for reporting and paying tax

### PR DESCRIPTION
**Pull request by grantrobson**
Can review but don't merge till Welsh content is received and integrated.

### CODE REVIEW

File: app/iht/views/deadlines.scala.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@import iht.utils._
+@import iht.constants.IhtProperties
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+
+@(implicit request:Request[_], messages: Messages, ihtFormPartialRetriever: FormPartialRetriever)
+    <p>@messages("page.iht.deadlines.p1")</p>
+    <p>@messages("page.iht.deadlines.p2") <a id="pay-early" href="@IhtProperties.linkPayEarly">@messages("page.iht.deadlines.anchorText")</a>.</p>
+    <p>@messages("page.iht.deadlines.p3")</p>
+
+    <a id="exit-to-iht400" class="button" href="@IhtProperties.linkEstateReportKickOut">@messages("page.iht.filter.paperform.million.exit")</a>
```
Comment (jennygrahamjones) [inline app/iht/views/deadlines.scala.html]: This should probably exit to the [Welsh IHT400 page](https://www.gov.uk/government/publications/inheritance-tax-inheritance-tax-account-iht400.cy) if the service is currently in Welsh

File: conf/messages.en [hunk 2]
```diff
@@ -2414,3 +2414,10 @@ label.beta.newservice =  This is a new service
 label.beta.improve =  will help us to improve it.
 label.beta.feedback = feedback
 label.beta.yours = - your
+
+# DEADLINES
+page.iht.deadlines.title = Deadlines for completing the IHT400 form
+page.iht.deadlines.p1 = You have 1 year from the date of death to fill in the IHT400 form if there is any Inheritance Tax to form.
```
Comment (jennygrahamjones) [inline conf/messages.en]: Last word should be pay, not form :)

Comment (grantrobson) [reply conf/messages.en]: Fixed!

File: app/iht/views/deadlines.scala.html (hunk 1 above)
Comment (grantrobson) [reply app/iht/views/deadlines.scala.html]: Fixed

**Pull request merged**